src/Constants.ts

```diff
@@ -232,10 +232,16 @@ export const ClassNames = {
     ELASTICSEARCH: 'ELASTICSEARCH',
     MYSQL: 'MYSQL',
     EVENTBRIDGE: 'AWS-EventBridge',
+    ZEIT: 'ZEIT',
 };
 
 export const AWS_SERVICE_REQUEST = 'AWSServiceRequest';
 
+export const ZeitTags = {
+    DEPLOYMENT_URL_HEADER: 'x-now-deployment-url',
```

Comment:
Can we have `ZeitConstants` and move `DEPLOYMENT_URL_HEADER` to there as `DEPLOYMENT_URL_HEADER` is not a tag but a constant

Reply:
👍